app/controllers/statistics_controller.rb

```diff
@@ -0,0 +1,34 @@
+class StatisticsController < ApplicationController 
+
+  def index
+    @cars = current_user.cars.order(created_at: :asc)  
+    
+    @total_repairs = Car.total_repairs
+    
+    @total_chargings = Car.total_chargings
+
+    @total = @total_repairs.merge(@total_chargings) { |_,a,b| a+b }
+
+    @all_car_repairs = []
+    @all_car_chargings = []
+    for car in @cars
+      if car.repairs.present?
+        @all_car_repairs.push(car.repairs.group_by_month(:date, format: "%B %Y").sum(:amount_cents))
+      end
+      if car.chargings.present?
+        @all_car_chargings.push(car.chargings.group_by_month(:date, format: "%B %Y").sum(:amount_cents))
+      end
+    end
+
+    @license_plate_repairs = []
```

Comment:
for example simply do: `@license_plate_repairs = @cars.pluck(:license_plate)`

Reply:
Ok thanks. I have done it. It's a lot easier with with queries and using scope in model. Simplicity sometimes is a lot better 😊